README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# pingmebot.PingMe project template
```

Comment:
I love the refactored changes but note that sometimes the refactoring may not always work properly, like in this example here. How can we avoid unwanted changes in future?